docs/src/plots/P3Melting.jl

```diff
@@ -1,15 +1,21 @@
-import CairoMakie as PL
-PL.activate!(type = "svg")
+using CairoMakie
```

Comment:
I prefer the `import CairoMakie as PL` way. Because it makes it clear in the code from which package the functions are coming from. But I'm not going to argue